climakitae/explore/vulnerability.py

```diff
@@ -365,21 +365,21 @@ def validate_params(self):
         # Validate time approach with Historical Reconstruction
         if (
             self.historical_data == "Historical Reconstruction"
-            and self.approach != "time"
+            and self.approach != "Time"
         ):
             errors.append(
                 "Historical Reconstruction data can only be retrieved using a time-based approach."
             )
 
         # Validating warming level inputs
-        if self.approach == "warming_level" and self.warming_level not in [
-            "1.5",
-            "2.0",
-            "2.5",
-            "3.0",
-            "4.0",
+        if self.approach == "Warming Level" and self.warming_level not in [
+            1.5,
+            2.0,
+            2.5,
+            3.0,
+            4.0,
```

Comment:
Is 4.0 an option now? If so, I'll amend the table again to include

Reply:
Let me remove that actually -- I'm not sure if that totally works with current CAVA design